### PR TITLE
Fixes ref key bindings

### PR DESCRIPTION
**Pull request by guillermooo**
@FichteFoll  PTAL

### CODE REVIEW

**Comment by guillermooo** [comment]
Just a handful of smaller edits, but I did some change some meanings slightly.

File: source/reference/key_bindings.rst [hunk 1]
```diff
@@ -321,20 +331,21 @@ International Keyboards
 
 Due to the way Sublime Text
 maps key names to physical keys,
-there might be a mismatch between the two.
+there might be a mismatch between the two
+in keyboards set to certain cultures.
```
Comment (FichteFoll) [inline source/reference/key_bindings.rst]: This doesn't make any sense. Or well, it might, but definitely poorly worded.

**Comment by FichteFoll** [comment]
otherwise lgtm

**Comment by guillermooo** [comment]
I'll squash all commits and resubmit this one again.

**Pull request closed without merging**